src/Domain/Creators/TimeDependentOptions/Python/ExpansionMap.cpp

```diff
@@ -0,0 +1,32 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Domain/Creators/TimeDependentOptions/Python/ExpansionMap.hpp"
+
+#include <array>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <string>
+#include <vector>
+
+#include "Domain/Creators/TimeDependentOptions/BinaryCompactObject.hpp"
+#include "Domain/Creators/TimeDependentOptions/ExpansionMap.hpp"
+
+namespace py = pybind11;
+
+namespace domain::creators::time_dependent_options::py_bindings {
+void bind_expansion_map(py::module& m) {
+  py::class_<time_dependent_options::ExpansionMapOptions<false>>(
+      m, "ExpansionMapOptions")
+      .def(py::init([](const std::array<double, 3>& initial_values_in,
```

Comment:
Also here and in the other classes: Use `py::init<...>()` so you don't have to define the lambda and deal with copying/moving the input params.